src/lib/logical_query_plan/import_node.hpp

```diff
@@ -13,13 +15,15 @@ namespace hyrise {
  */
 class ImportNode : public EnableMakeForLQPNode<ImportNode>, public AbstractNonQueryNode {
  public:
-  ImportNode(const std::string& init_table_name, const std::string& init_file_name, const FileType init_file_type);
+  ImportNode(const std::string& init_table_name, const std::string& init_file_name, const FileType init_file_type,
+             const std::optional<EncodingType> init_table_encoding = std::nullopt);
```

Comment:
The ctor is not used _that_ often in the codebase, so I'd vote against a default argument and for adapting these cases. As we often use `::make()` with specifying the input(s), we have to pass an argument anyways.